documentation/adr/template.md

```diff
@@ -0,0 +1,68 @@
+# {short title of solved problem and solution}
+
+- **Status:** {Draft | Proposed | Rejected | Accepted | Deprecated | ... | Superseded By [xxx](yyyymmdd-xxx.md)} <!-- REQUIRED -->
+- **Last Modified:** {YYYY-MM-DD when the decision was last updated} <!-- REQUIRED -->
+- **Related Issue:** [#{issue number}](https://github.com/HHS/grants-api/issues) <!-- RECOMMENDED -->
+- **Deciders:** {list everyone involved in the decision} <!-- OPTIONAL -->
+- **Tags:** {space and/or comma separated list of tags} <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+{Describe the context and problem statement, e.g., in free form using two to three sentences. You may want to articulate the problem in form of a question.}
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- {driver 1, e.g., a constraint, priority, condition, etc.}
+- {driver 2, e.g., a constraint, priority, condition, etc.}
+- ...
+
+## Options Considered
+
+- {option 1}
+- {option 2}
+- ...
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: "{option 1}", because {justification. e.g., only option which meets k.o. criterion decision driver | which satisfies x condition | ... }.
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- {e.g., improved performance on quality metric, new capability enabled, ...}
+- ...
+
+### Negative Consequences <!-- OPTIONAL -->
+
+- {e.g., decreased performance on quality metric, risk, follow-up decisions required, ...}
+- ...
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### {option 1}
+
+{example | description | pointer to more information | ...} <!-- OPTIONAL -->
+
+- **Pro**
+  - Good, because {argument a}
+  - Good, because {argument b}
+  - ...
+- **Cons**
+  - Bad, because {argument c}
+  - ...
+
+### {option 2}
+
+{example | description | pointer to more information | ...} <!-- OPTIONAL -->
+
+- **Pro**
+  - Good, because {argument a}
+  - Good, because {argument b}
+  - ...
+- **Cons**
+  - Bad, because {argument c}
+  - ...
+
+## Links <!-- OPTIONAL -->
+
+- [{Link type}](link to adr) <!-- example: Refined by [xxx](yyyymmdd-xxx.md) -->
```

Comment:
nit: i think this example is copy/pasta from a previous line?

Reply:
Yeah good call! I meant this more for external resources that might be related to the ADR. I'll update this line to be:
```suggestion
- [{Link name}](link to external resource)
```